cli/ballerina-launcher/src/main/resources/cli-help/ballerina-init.help

```diff
@@ -12,8 +12,11 @@ DESCRIPTION
      sample Ballerina source file. You can use Ballerina.toml to manage
      the organization setting and version for the project.
      
-     The generated set of artifacts also includes a service skeleton 
-     and a test skeleton that you can use to kick start the project. 
+     If you run the init command in the interactive mode (--i) and
+     create a package with a service or a main function, then the
+     generated set of artifacts includes a service skeleton or a main
+     function skeleton depending on if its a service or a main function
+     and a test skeleton that you can use to kick start the project.
```

Comment:
Does this part sound redundant to say ? 

' depending on if its a service or a main function and'

Reply:
Yes this sounds redundant so removed that part with 1cdb231